app/http.php

```diff
@@ -643,6 +641,10 @@ function removeAllRuntimes(Table $activeRuntimes, Orchestration $orchestration):
 
             $activeRuntimes->del($runtimeName);
 
+            $error = empty($output)
+                ? 'Failed to execute build.'
+                : "Failed to execute build command:\n\n" . $output;
```

Comment:
One linebreak should be enough

```suggestion
                : "Failed to execute build command:\n" . $output;
```